src/components/App.tsx

```diff
@@ -588,6 +618,11 @@ export class App extends React.Component<AppProps, AppState> {
     </Split>;
 
     return <div className="fill">
+      {this.state.toast &&
```

Comment:
I vote for using
```
if(this.state.toast) {
  return <Toast />
}
```
Or something like that. `&&` looks confusing.

Reply:
Using `&&` is the JSX way of doing this.